src/table/utils/handle-key-press.ts

```diff
@@ -54,35 +54,42 @@ export const getNextCellCoord = (
   evt: React.KeyboardEvent,
   rootElement: HTMLElement,
   cellCoord: [number, number],
-  topAllowedRow = 0
+  allowedRows: {
+    selected: {
+      cellIdx: number;
+      bottom: number;
+    };
+    cellIdx: number;
+    bottom: number;
+  } = { selected: { cellIdx: 0, bottom: 0 }, cellIdx: 2, bottom: 1 }
```

Comment:
It went confusing so I decided to set the object with both selected and without selection cell index. 😣